src/views/Home/index.js

```diff
@@ -90,7 +93,9 @@ const Home = ({ authentication, onLogIn }) => {
   } // @WELLNESS-CHECK disabled to revert this you must uncomment this lines of code
   //else if (networkStatus === 'online' && !hasAnswered) {
   //return <WellnessQuestion setStatus={() => setHasAnswered(true)} />;}
-  else {
+  else if (!checkedIn && personType.includes('stu')) {
+    window.location.replace("AcademicCheckIn")
```

Comment:
Please use `react-router` hooks to navigate, rather than `window.location.replace`.
```javascript
let history = useHistory()
...
history.replace("AcademicCheckIn")
```